project_proposal/tex/mainpage.tex

```diff
@@ -0,0 +1,376 @@
+\label{figure:system_drawing}
+\caption{System drawing} 
+\end{figure}
+
+Our distributed system consists of five sub-systems, where we are writing three
+of them.
+
+\section{OPC-UA Client}
+The OPC-UA client is responsible for communicating with the brewing machine via
+the OPC-UA server. This is needed in order to send and receive data via
+commands.
+
+\section{Website}
+The website user interface is where the client will be able to interface with
+our system.
+
+\section{REST API}
+The REST API functions primarily as a gateway in this design, in order for the
+front-end to receive data via fetch calls in JavaScript it needs to talk with
+something that speaks HTTP and preferably delivers data in JSON.
+
+\section{Objectives}
+The group wants to create a dashboard solution, that gets the necessary
+information from the brewery machine to display information about current - and
+previous batches.
+
+The group wants to host the REST API and Website on a Linux based server on a
+fully-qualified domain name using the container technology Docker which should
+be configured via Docker Compose. Furthermore the group aims to setup a
+continuous integration pipeline for development via GitHub Actions and
+continuous delivery via Watchtower.
+
+\section{Initial Requirement Analysis}
+
+
+\subsection{Summary of requirements}
+The groups proposed solution will adhere to the requirements given by the brewery Refslevbæk Bryghus A/S.
+
+The MES must be able to control the brewery’s production.
+It must be able to start and stop the production line,
+as well as monitor the production and collect data from the production line.
+The data must be stored for further analysis.
+The MES must be able to keep tract of the batches that the new machine is producing,
+as well as collect various data from the machine that is associated with the current batch number.
+After a finished batch production, the MES must be able to produce a batch report of the produced batch.
+The report must contain the following. \\
+
+\begin{itemize}
+    \item this Batch ID
+    \item Product type
+    \item Amount of products (total, defect and acceptable)
+    \item Amount of time used in the different states
+    \item Logging of temperature over the production time
+    \item Logging of timidity over the production time
+\end{itemize}
+
+The MES/SCADA system must be able to monitor the production and display live relevant data from the machine.
+The documentation of the system must contain an illustration that defines the different components in the setup, in relation to the ISA88^1213 Part 1 Physical Hierarchy model. The system must have a visualization that can be accessed and used to display the production data. The system must be able to collect the necessary data from the machine and calculate the OEE^131516 of the machine. The OEE must be available to be displayed by the system. The system must be able to estimate the error function associated with the different products.
```

Comment:
This line is broken. There needs to be a backslash at '^', at ISA88^1213.

Reply:
I am not sure I escaped it, but that just put the next character like this 
![image](https://user-images.githubusercontent.com/55736468/94797437-665f2080-03e0-11eb-8f24-86867e82e55b.png)

then i did this instead ISA88\^{}1213
![image](https://user-images.githubusercontent.com/55736468/94797547-90b0de00-03e0-11eb-8a30-f2fa975aab6e.png)

so maybe it is resolved you let me know?

Reply:
@Kechr

Reply:
Looks good.